app/views/certificates.html

```diff
@@ -122,6 +122,8 @@
 
               <!-- Server Certificate Alerts -->
               <alert ng-repeat="alert in alerts.serverKey" type="alert.type" style="margin: 20px 0px 0px 0px;">{{alert.msg}}</alert>
+              <h4><strong>Enter a passphrase (Optional)</strong></h4>
+              <input class="form-control" placeholder="Enter a passphrase here if the key requires one." type="text" ng-model="serverPassphrase" />
```

Comment:
Adding a symbol will not be possible without making changes to the core. I would need to return the whole keystore to get the passphrase bit